### ISSUE TITLE
Is the Minwaita theme ABANDONED?

### ISSUE THREAD

**Issue by corebots** [godlyranchdressing/Minwaita#39]

**Comment by 6wl** [comment]
Minwaita has been my go to theme to make Gnome3 usable in terms of screenspace.  Would be sad to to see it die.

Worst case is there other alternatives that are maintained?

**Comment by corebots** [comment]
Haven't been able to find any better alternative myself, still using Minwaita till present.

**Comment by godlyranchdressing** [comment]
No, I've just been preoccupied with other projects. Online and off. Also, the truth is I kind of rushed into this all without knowing much about directory structure, git and building, so, I'm thinking to overhaul everything to make maintenance and keeping up with GNOME updates easier. I just can't say when I'll get time to do that. My sincerest apologies. 

There is [Adwaita-Slim](https://github.com/godlyranchdressing/Adwaita-Slim) which is probably more update-friendly since it's just Adwaita with less padding.
